docker/entrypoint.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+if [ -n "$ZTAUTH" ] && [ "$ZTAUTH" != "''" ]; then
```

Comment:
Hi @PassiveLemon , I am a bit concern about this implementation here. That `authtoken.secret` should be automatically generated by zerotier-one which is also used to generate the node's UUID within the network. I am not sure what is the effect of this implementation (writing to `authtoken.secret` into zerotier system file, instead of letting zerotier generate it automatically) will cause any runtime issue with multiple Zoraxy docker nodes in the same network.

Reply:
@PassiveLemon Yes, you understand it correctly. The variable is there just in case Zoraxy do not have permission to read the `authtoken.secret` file but the user still want to use ZeroTier function via API calls to the zerotierd daemon.

Reply:
Okay, I will make the appropriate changes

Reply:
I have the auto-generated part done, it picks up the token and allows network creation. However I was testing to see what happens if I do provide a ZTAUTH token, and when I do, I get a permission error. Not sure what the expected behavior is.